### ISSUE TITLE
Review Ticket-Khushi

### ISSUE THREAD

**Issue by KhushiB24** [sonakshibhalla/tri2_turtles.github.io#125]
Tutorials:
-attend tutorials when needed to catch up with deployment and the weekly projects.
Individual Runtime review on About Page progress/quality.
Individual Analytics: 
-[Stress Relief Page](https://github.com/sonakshibhalla/tri2_turtles.github.io/commit/510f67f8fa564d0464c6ce955b0b1098b2105c01)
CB progress:
-[work on practice tests](https://docs.google.com/document/d/14XqjwIx9GVl1AVqknS8_d6Nvgu2xOHg0OkKLqV-oSxk/edit?usp=sharing)
-Deployment Certbot
[update and deploy website](https://docs.google.com/document/d/1-OsHEh9xUjh7hq7cAVZrLJoxCZHEh60w8b4kT-KPHJI/edit?usp=sharing)
Databases:
-[working on article API](https://rapidapi.com/offpagepamllc@gmail.com/api/psych-associates-of-maryland/)

**Comment by KhushiB24** [comment]
Self-score:
1) Attendance and tutorial: 0.75
-explanation: I have been attending tutorial for additional help regarding the deployment process as well as the team projects that we are working on.
2) Individual Runtime: 0.5
-explanation: This week I made less commitments towards the my About page, however I had made plans as to what updates would be made.
3) Individual Analytics: 0.5
-explanation: This week I worked on the stress relief page, where the major contribution would be able to figure out the color of the button as well as the font of the button.
4) CB Progress: 1
-explanation: for further practice, I have been practicing MCQ questions
5) Deployment Certbot and Databases: 0.5
-explanation: I have been adding more technical, where I plan to create an article API based on our team idea: Mental Health Article API
